rotkehlchen/db/filtering.py

```diff
@@ -62,18 +62,22 @@ class DBTimestampFilter(DBFilter):
     from_ts: Optional[Timestamp] = None
     to_ts: Optional[Timestamp] = None
     scaling_factor: Optional[FVal] = None
+    timestamp_field: Optional[str] = None
 
     def prepare(self) -> Tuple[List[str], List[Any]]:
         filters = []
         bindings = []
+        timesatmp_field = 'timestamp'
```

Comment:
Here and everywhere you use it

```suggestion
        timestamp_field = 'timestamp'
```